app/components/app_patient_medical_history_card_component.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="nhsuk-card" id="consent">
```

Comment:
Is id="consent" correct? Is it used for anything?

Reply:
Pretty sure it isn't. I think it was being used for testing before but I switched it up. Kheel it?